tests/unit/infrastructure/ai/providers/test_anthropic_provider.py

```
@@ -10,6 +10,12 @@
 class TestAnthropicProvider:
     """AnthropicProvider 测试"""
 
+    @pytest.fixture(autouse=True)
+    def clear_proxy_env(self, monkeypatch):
+        """隔离宿主机代理环境，避免污染测试。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)